src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -61,13 +74,45 @@ function CardSection() {
                                     })}
                                 </Text>
                             </View>
+                            <CardSectionActions />
                         </View>
-                        <CardSectionActions />
-                    </>
+                    )}
+                    {isEmptyObject(defaultCard?.accountData) && <CardSectionDataEmpty />}
+                </View>
+                {!isEmptyObject(defaultCard?.accountData && plan && isEligibleForRefund) && (
+                    <MenuItem
```

Comment:
What happens if the API throws an error. Don’t we need to show any RBR here ?

Reply:
As we are using Pattern C in that flow , it shouldnt be available

Reply:
there was nothing about that in doc, @blimpich what do you think ?

Reply:
For this PR lets not worry about it. We'll deal with error cases for this later.

Reply:
thanks for confirming.